src/pybind/mgr/dashboard/frontend/src/styles/defaults.scss

```diff
@@ -207,3 +207,7 @@ $health-chart-colors: (
   'cyan': #73c5c5,
   'purple': #3c3d99
 );
+
+.text-warning-dark {
+  color: #fd7e14;
```

Comment:
In octopus, the 
```
 $theme-colors: (
       'primary': $color-primary,
      .....
    ) 
```
is actually placed inside the styles.scss. Maybe `'warning-dark': #fd7e14` would fit inside the `$theme-colors` in styles.scss? @callithea

Reply:
Thanks for the suggestion, @nizamial09. @Exotelis what do you think?

Reply:
Fine with me, but do it like this please:

```
$fa-font-path: '~fork-awesome/fonts';
$font-family-icon: 'ForkAwesome';

// Bootstrap
$warning-dark: #fd7e14;
$theme-colors: (
  'primary': $color-primary,
  'secondary': $color-accent,
  'dark': $color-mild-gray,
  'warning-dark': $warning-dark
);
```

The class definition should then be obsolete ;)

Reply:
updated :)